runtime/autoload/health/provider.vim

```diff
@@ -53,7 +53,7 @@ function! s:check_clipboard() abort
   if empty(clipboard_tool)
     call health#report_warn(
           \ "No clipboard tool found. Using the system clipboard won't work.",
-          \ ['See ":help clipboard"'])
+          \ ['See |clipboard|.'])
```

Comment:
I guess this will happen more often in the future. Should I create a `health#help_to_link()` for this purpose?